src/cryptography/hazmat/backends/interfaces.py

```diff
@@ -317,6 +317,18 @@ def x509_name_bytes(self, name):
         Compute the DER encoded bytes of an X509 Name object.
         """
 
+    @abc.abstractmethod
+    def load_pem_x509_crl(self, data):
```

Comment:
```suggestion
    def load_pem_x509_crl(self, data: bytes) -> "CertificateRevocationList":
```

Reply:
Doing this will cause an import loop so I'd prefer figuring out how to type it in a separate PR :)